src/ansys/mapdl/core/xpl.py

```diff
@@ -36,7 +35,9 @@
 
 def id_generator(size=6, chars=string.ascii_uppercase):
     """Generate a random string using only uppercase letters."""
-    return "".join(random.choice(chars) for _ in range(size))
+    import secrets
+
+    return "".join(secrets.choice(chars) for _ in range(size))
```

Comment:
This was a perfect opportunity to remove a duplicated function .. this one or ``random_string``.

Reply:
Done in [30aae0a](https://github.com/ansys/pymapdl/pull/3474/commits/30aae0a7c803eb2a383324c1105b2b0bd1eef787)